api_shield_api_discovery.go

```diff
@@ -0,0 +1,164 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/goccy/go-json"
+)
+
+// APIShieldDiscoveryOperation is an operation that was discovered by API Discovery.
+type APIShieldDiscoveryOperation struct {
+	// ID represents the ID of the operation
+	ID string `json:"id"`
```

Comment:
Should this not be a uuid?

Reply:
have answered in https://github.com/cloudflare/cloudflare-go/pull/1413#discussion_r1346195655